javascript/695-max-area-of-island.js

```diff
@@ -2,32 +2,35 @@
  * @param {number[][]} grid
  * @return {number}
  */
- var maxAreaOfIsland = function(grid) {
```

Comment:
This file has a conflicting issue due to main have 2 files with lowercase and uppercase